README.md

```diff
@@ -212,7 +214,83 @@ transloadit.assembly(
 
 Read up more on the notifications [on Transloadit's documentation page](http://transloadit.com/docs/notifications-vs-redirect-url)
 
-### 7. Getting Bill reports
+### 7. Other Assembly methods
+
+Transloadit also provides methods to retrieve/replay assemblies and their notifications.
+
+```ruby
+assembly = transloadit.assembly
+
+# returns a list of all assemblies
+assembly.list
+
+# returns a specific assembly
+assembly.get 'YOUR_ASSEMBLY_ID'
+
+# replays a specific assembly
+response = assembly.replay 'YOUR_ASSEMBLY_ID'
+# should return true if assembly is replaying and false otherwise.
+response.replaying?
+
+# returns all assembly notifications
+assembly.get_notifications
+
+# replays an assembly notification
+assembly.replay_notification 'YOUR_ASSEMBLY_ID'
+```
+
+### 8. Templates
+
+Transloadit provides a [templates api](https://transloadit.com/docs/templates)
+for recurring encoding tasks. Here's how you would create a template:
+
+```ruby
+template = transloadit.template
+
+# creates a new template
+template.create(
+  :name => 'TEMPLATE_NAME',
+  :template => {
+    "steps": {
+      "encode": {
+        "use": ":original",
+        "robot": "/video/encode",
+        "result": true
+      }
+    }
+  }
+)
+```
+
+There are also some other methods to retrieve, update and delete a template.
+
+```ruby
+# returns a list of all templates.
+template.list
+
+# returns a specific template.
+template.get 'YOUR_TEMPLATE_ID'
+
+# updates the template whose id is specified.
+template.update(
+  'YOUR_TEMPLATE_ID',
+  :name => 'CHANGED_TEMPLATE_NAME',
+  :template => {
+    "steps": {
+      "encode": {
```

Comment:
@ifedapoolarewaju what @einzige said, we can remove the quotes here, less syntax clutter :)